lib/myfinance/resources/financial_account.rb

```diff
@@ -28,10 +26,33 @@ def create(entity_id, params = {})
       #   Documentation: https://app.myfinance.com.br/docs/api/receivable_accounts#delete_destroy
       #
       def destroy(id, entity_id)
-        http.delete(endpoints(id, entity_id, :destroy)) do |response|
+        http.delete(endpoint_for(id, entity_id, :destroy)) do |response|
```

Comment:
Unused block argument - `response`. You can omit the argument if you don't care about it.